README.md

```diff
@@ -86,39 +86,7 @@ The `import.js` file you provide will be bundled with any locally referenced scr
 
 ### Assistant
 
-Run AI-enabled commands to assist with your import script development.
-
-Add an npm script entry to your Edge Delivery project's `package.json`:
-
-```
-"assistant": "aem-import-helper assistant"
-```
```

Comment:
I would leave this section in the README that explains how to add the assistant to a project. Then link to the doc page that goes into detail about each command.